File: api/RS.API/RS.Entity/Models/ApprovalEventRoles.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RS.Entity.Models
{
    public partial class ApprovalEventRoles : BaseEntity
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity), Key]
        public int ApprovalEventRoleId { get; set; }
        public int ApprovalEventId { get; set; }
        public int RoleId { get; set; }

        public ApprovalEvents ApprovalEvent { get; set; }
        public Roles Role { get; set; }

        public Guid UserId { get; set; }
        public Users User { get; set; }
    }
}

File: web/src/app/webapi/models/approval-event-roles.ts
/* tslint:disable */
import { ApprovalEvents } from './approval-events';
import { Roles } from './roles';
import { Users } from './users';

/**
 */
export class ApprovalEventRoles {
    isActive?: boolean;
    approvalEventRoleId?: number;
    roleId?: number;
    approvalEvent?: ApprovalEvents;
    role?: Roles;
    userId?: string;
    user?: Users;
    approvalEventId?: number;
    createdBy?: string;
    createdDate?: string;
    modifiedBy?: string;
    modifiedDate?: string;
    isDeleted?: boolean;
    deletedBy?: string;
    deletedDate?: string;
}
